### DISCUSSION TITLE
celery_sqlalchemy_scheduler does not run at all

### DISCUSSION THREAD

**Discussion by aliscie** [sqlalchemy/sqlalchemy#7214] [Usage Questions]
.

**Comment by CaselIT** [comment]
Hi,

I don't think this is the best place to seek help, since sqlalchemy is not involved on with either of these projects. I've never used them so I'm not sure what's up.

**Reply by CaselIT** [reply]
btw, you can leave the original question, maybe other members of the community can offer help
